pkg/virt-launcher/virtwrap/device/hostdevice/sriov/pcipool_test.go

```diff
@@ -162,7 +163,7 @@ func newNetworkData(netName string, resource resourceData) networkData {
 		Name: netName,
 		ResourceEnv: envData{
 			"KUBEVIRT_RESOURCE_NAME_" + netName,
-			"intel.com/" + resource.Name + "_pool",
+			path.Join("intel.com", resource.Name+"_pool"),
```

Comment:
This is not a file path or an URL, I see no added value here to use this `Join`.

Reply:
This has been changed in the first commit and then reverted in a following commit.
Please perform a rebase and drop this change from the first commit.

I saw this on other changes as well.

Reply:
Done